doc/source/glossary.rst

```diff
@@ -555,56 +483,79 @@ Glossary
+
+        >>> dt = np.dtype([('a', np.int32), ('b', np.float32, (3,))])
+        >>> np.zeros(3, dtype=dt)
+        array([(0, [0., 0., 0.]), (0, [0., 0., 0.]), (0, [0., 0., 0.])],
+              dtype=[('a', '<i4'), ('b', '<f4', (3,))])
+
 
 
    subarray data type
-       A :term:`structured data type` may contain a :term:`ndarray` with its
-       own dtype and shape:
+      An element of a structured datatype that behaves like an ndarray.
+
+      ..
 
-       >>> dt = np.dtype([('a', np.int32), ('b', np.float32, (3,))])
-       >>> np.zeros(3, dtype=dt)
-       array([(0, [0., 0., 0.]), (0, [0., 0., 0.]), (0, [0., 0., 0.])],
-             dtype=[('a', '<i4'), ('b', '<f4', (3,))])
 
 
    title
-       In addition to field names, structured array fields may have an
-       associated :ref:`title <titles>` which is an alias to the name and is
-       commonly used for plotting.
+       An alias for a field name in a structured datatype.
 
 
    ufunc
-       Universal function.  A fast element-wise, :term:`vectorized
-       <vectorization>` array operation.  Examples include ``add``, ``sin`` and
-       ``logical_or``.
+       NumPy's fast element-by-element computation (:term:`vectorization`) is
+       structured so as to leave the choice of function open. A function used
+       in vectorization is called a ``ufunc``, short for ``universal
+       function``. NumPy routines have built-in ufuncs, but users can also
+       :doc:`write their own. <reference/ufuncs>`
+
 
 
    vectorization
-       Optimizing a looping block by specialized code. In a traditional sense,
-       vectorization performs the same operation on multiple elements with
-       fixed strides between them via specialized hardware. Compilers know how
-       to take advantage of well-constructed loops to implement such
-       optimizations. NumPy uses :ref:`vectorization <whatis-vectorization>`
-       to mean any optimization via specialized code performing the same
-       operations on multiple elements, typically achieving speedups by
-       avoiding some of the overhead in looking up and converting the elements.
+       NumPy hands off array processing to C, where looping and computation are
+       much faster than in Python. To exploit this, programmers using NumPy
+       eliminate Python loops in favor of array-to-array operations.
+       :term:`vectorization` can refer both to the C offloading and to
+       structuring NumPy code to leverage it.
+
 
 
    view
-       An array that does not own its data, but refers to another array's
-       data instead.  For example, we may create a view that only shows
-       every second element of another array::
+       Without changing underlying data, NumPy can make one array masquerade as
+       any number of other arrays with different types, shapes, and even
+       content. This is much faster than creating those arrays.
+
+       An array created this way is a ``view``, and the performance gain often
```

Comment:
```suggestion
       An array created this way is a "view", and the performance gain often
```
This isn't really a code snippet

Reply:
[Google style](https://developers.google.com/style/key-terms) for this is italics, will change.

Reply:
Works for me, I can just never remember the rst syntax